source_contrasts/source_contrasts.py

```diff
@@ -203,6 +208,11 @@ def process_one_subject(
 
             # Plot the morphed brain
             brain = plot_stc(stc=stc_contrast_morphed, fs_subjects_dir=fs_subjects_dir)
+            # add text to the plot
+            title_string = (f"{contrast.cond_1}–{contrast.cond_2} \n"
+            f"{freq_band.fmin}–{freq_band.fmax} Hz \n"
+            f" {TMIN}–{TMAX} s \n")
```

Comment:
Nice! Do we need the `\n` here in the last line? And the leading space? The trailing spaces we can safely omit, I believe

```suggestion
            title_string = (f"{contrast.cond_1}–{contrast.cond_2}\n"
            f"{freq_band.fmin}–{freq_band.fmax} Hz\n"
            f"{TMIN}–{TMAX} s")
```